View/AppStartPoint.cpp

```diff
@@ -0,0 +1,73 @@
+#include "ViewInitializer.h"
+#include "StatisticsDisplayer.h"
+#include <string>
+#include <iostream>
+#include <GL/glut.h>
+#include <GL/glew.h>
+using namespace MeshIS::View;
+
+vector<function<void()>> renderFunctions;
```

Comment:
What is rationale behind this global? See: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#i2-avoid-global-variables